tensorbay/opendataset/_utility/voc.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Get_vocdetection_data method for open dataset."""
```

Comment:
No need to be too specific. Would `Common methods for loading VOC formatted datasets.` do?